Add function to allow easy statusline integration (#11)
kosayoda/nvim-lightbulb: README.md

        enabled = false,
        -- Text to show at virtual text
        text = "💡",
    },
    status_text = {
        enabled = false,

>         -- Text to show at when status text requested
>         text = "💡",
>         -- Text to show when status text requested and no actions available
>         not_lit_text = ""

Reviewer: ```suggestion
        -- Text to provide when code actions are available
        text = "💡",
        -- Text to provide when no actions are available
        text_unavailable = ""
```
